homework-2/js/main.js

```diff
@@ -0,0 +1,92 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.hamburger-menu');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu-body');
+    iconMenu.addEventListener('click', function(e){
+      document.body.classList.toggle('lock');
+      iconMenu.classList.toggle('active');
+      menuBody.classList.toggle('active');
+    });
+  }
```

Comment:
This is not enough - try open menu and resize browser, so open menu button will disappear - site will be broken.

Reply:
done